### ISSUE TITLE
Improve usability of "ignore issues" feature

### ISSUE THREAD

**Issue by sebastiansirch** [viadee/vPAV#73]
Currently, ignored issues can only be collected via the diagram. It would be handy to have the same button in the table as well as a download-button at the buttom.

We could even further enhance this feature by showing a list of collected issues with the ability to remove specific issues from the collection. (Currently, it is not possible to remove an issue after it has been added once).

**Comment by sebastiansirch** [comment]
Another aspect: Lines in the ignoredIssues.txt file are cut off after 32 chars which makes this file very hard to read. I don't see a reason why we need to shorten the lines that heavily. 

Example where it is hard to guess the meaning...
```
#Element mit ID 'serviceTask_d..
123somehashvalues456
#Element mit ID 'serviceTask_a..
123anotherhashvalues456
```

**Comment by fkoehne** [comment]
I made separate issue out of this one: #74.

**Comment by sdibernardo** [comment]
I agree on the usability of having a button in the table to add issues to the whitelist. Will keep this in mind. Thanks for the feedback

**Comment by sdibernardo** [comment]
Added some buttons and functionality. Please have a look. Im open for feedback :)
@sebastiansirch 
@fkoehne 
@cl-a-us
